.circleci/config.yml

```diff
@@ -274,7 +274,7 @@ jobs:
                 no_output_timeout: 240m
                 command: |
                   source ~/.bashrc
-                  tox -v -e << parameters.toxenv >> -- << parameters.tox_args >>
+                  tox r -v -e << parameters.toxenv >> -- << parameters.tox_args >>
```

Comment:
Not a request for a change, just noting that I have a personal preference for long rather than short options in scripts so that it's easier for someone unfamiliar with the tool to see what's going on; so here I would write `tox run ...` (`-v` for verbose is common enough to abbreviate).